adr/2025-10-28-changelog-release-info-process.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Replace changelog ADR with "Changelog & Release Info Process"
+date: 2025-10-28
+area: process
+tags: [documentation, release, changelog, release-info, upgrade, developer-relations]
+authors: [Álvaro Thomas, Jonas Elfering]
+status: accepted
+---
+
+## Context
+
+Historically, Shopware used a changelog file workflow based on per-change markdown files under `/changelog/_unreleased` with an automated build step that aggregated them into `CHANGELOG.md` and `UPGRADE.md`.  
+While technically useful, this caused friction: duplication, missing high-level developer-facing context, and the release notes living in a separate repository.  
+The team now prefers a curated, in-repo workflow for developer-facing release notes.
+
+---
+
+## Decision
+
+1. The ADR `2020-08-03-implement-new-changelog.md` is **superseded** and archived under `adr/_superseded/`.
+2. We adopt a **curated in-repo model**:
+   - `RELEASE_INFO-6.<currentMajor>.md`: developer-facing release notes with per-minor sections.
+   - `UPGRADE-6.<upcomingMajor>.md`: upgrade instructions and breaking changes.
+```
+Example: RELEASE_INFO-6.7.md
+
+## 6.7.5 (Upcoming)
+### Core
+- Added new `CartRoute` parameter `guestLogin` (PR #12326)
+- Improved indexing performance for product categories (PR #12657)
+### Administration
+- Updated TypeScript target to ES2023 (PR #12408)
+``` 
+```
+Example: UPGRADE-6.8.md
+
+## Break: Deprecated SalesChannelContextSwitcher
+- **What changed:** The old service is deprecated and will be removed in 6.8.
```

Comment:
Do we really want to predefine those three bullet points? or would be sentences enough?